go.mod

```diff
@@ -26,27 +26,28 @@ require (
 	github.com/prometheus/client_golang v1.16.0
 	github.com/prometheus/common v0.44.0
 	github.com/spf13/cobra v1.7.0
-	github.com/stretchr/testify v1.8.2
+	github.com/stretchr/testify v1.8.4
 	github.com/summerwind/h2spec v0.0.0-20200804131034-70ac22940108
 	github.com/tcnksm/go-httpstat v0.2.1-0.20191008022543-e866bb274419
 	go.uber.org/zap v1.25.0
+	golang.org/x/exp v0.0.0-20220827204233-334a2380cb91
 	golang.org/x/time v0.3.0
-	google.golang.org/api v0.114.0
-	google.golang.org/grpc v1.54.0
+	google.golang.org/api v0.126.0
+	google.golang.org/grpc v1.58.3
 	gopkg.in/fsnotify.v1 v1.4.7
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.28.3
-	k8s.io/apiextensions-apiserver v0.28.3
-	k8s.io/apimachinery v0.28.3
-	k8s.io/apiserver v0.28.3
+	k8s.io/api v0.29.0
```

Comment:
Could we go ahead and take 0.29.1 for these K8S packages?

Reply:
This PR only bumps the k8s.io/* packages to v0.29.0 because the openshift/api bump requires doing so.  I expect we'll bump to k8s.io/api v0.29.1 or later in a future PR (for example, for a controller-runtime bump or another openshift/api bump).

Reply:
That's fine if we want do it later. It was more of a "why not use the latest if we are bumping now" sort of thing, but we can just plan on doing it later.